rules/lfi-os-files.data

```diff
@@ -1450,3 +651,125 @@ sys/hypervisor
 sys/kernel
 sys/module
 sys/power
+system/library/webobjects/adaptors
+system32/config
+system32/inetsrv/config
+tmp/access.log
+tmp/kafka-logs
+tsconfig.json
+typo3conf/localconf.php
+usr/apache/conf
+usr/apache2/conf
+usr/etc/pure-ftpd.conf
+usr/home/user/lighttpd
+usr/home/user/var/log
+usr/internet/pgsql/data
+usr/lib/cron/log
+usr/lib/php.ini
+usr/lib/php
+usr/lib/rpm/rpm.log
+usr/lib/security
+usr/local/zeus/web
+usr/pkg/etc/httpd
+usr/pkgsrc/net/pureftpd
+usr/ports/contrib/pure-ftpd
+usr/ports/ftp/pure-ftpd
+usr/sbin/mudlogd
+usr/sbin/mudpasswd
+usr/sbin/pure-config.pl
+usr/share/adduser/adduser.conf
+usr/share/logs
+usr/share/squirrelmail
+usr/share/tomcat
+usr/spool/lp/log
+usr/spool/mqueue/syslog
+var/adm/acct/sum/loginlog
+var/adm/aculog
+var/adm/crash
+var/adm/cron/log
+var/adm/dtmp
+var/adm/lastlog/username
+var/adm/log
+var/adm/lp/lpd-errs
+var/adm/messages
+var/adm/pacct
+var/adm/qacct
+var/adm/ras
+var/adm/sulog
+var/adm/syslog
+var/adm/utmp
+var/adm/vold.log
+var/adm/wtmp
+var/adm/x0msgs
+var/apache/conf/httpd.conf
+var/apache/logs
+var/apache2/config.inc
+var/cpanel
+var/cron/log
+var/data/elasticsearch
+var/data/mysql-bin
+var/htmp
+var/lib/elasticsearch
+var/lib/mysql
+var/lib/pgsql
+var/lib/squirrelmail
+var/lighttpd
+var/local/www/conf
+var/log
+var/lp/logs
+var/mail
+var/mysql-bin
+var/mysql.log
+var/nm2/postgresql.conf
+var/postgresql
+var/run/utmp
+var/saf/_log
+var/saf/port/log
+var/spool/cron/crontabs/root
```

Comment:
`spool` is probably also something we could just block entirely.

Reply:
Blocking `var/spool`.